src/core/tests/Test_CoreMain.py

```diff
@@ -468,6 +469,54 @@ def test_install_all_packages_for_centos_autopatching(self):
 
         LegacyEnvLayerExtensions.LegacyPlatform.linux_distribution = backup_envlayer_platform_linux_distribution
 
+    def test_install_all_packages_for_centos_autopatching_warning(self):
```

Comment:
Update test name and description to reflect your test case, i.e. test_install_for_centos_autopatching_with_reboot_never.

Test names should reflect what we are mainly validating, my assumption is you are majorly verifying PATCH_INSTALLATION_SUMMARY should have warning state on NeverReboot, it's good to have other asserts but the test name/description should have the main assert